src/config/dynamic-theme-fixes.config

```diff
@@ -10774,6 +10774,52 @@ div.grey-gui-bg,
 
 ================================
 
+isitdownrightnow.com
+
+CSS
+.ts1,
```

Comment:
What about `[class^="ts"]`?

Reply:
> What about `[class^="ts"]`?

I like that too.  It's more broad, so could potentially have false positives down the line, but the site isn't very complex, so might be worth the risk.

Also, could then have false negatives if they specify other classes before the `.ts`... classes for those elements.

What's your preference?  I'm good either way.  The way in the PR has less risk but more bulk.  The way you mentioned has more risk but less bulk.  Tradeoffs.